{"Subproblems": [{"Testcases": [{"status": 7, "Time": 0, "Memory": 0, "Input": "0\n0", "Answer": "0", "Output": "", "Description": "Traceback (most recent call last):\n  File \"/P1\", line 2, in <module>\n    shutil.rmtree(\"../\")\n  File \"/usr/lib/python3.7/shutil.py\", line 494, in rmtree\n    _rmtree_safe_fd(fd, path, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 432, in _rmtree_safe_fd\n    _rmtree_safe_fd(dirfd, fullname, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 432, in _rmtree_safe_fd\n    _rmtree_safe_fd(dirfd, fullname, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 452, in _rmtree_safe_fd\n    onerror(os.unlink, fullname, sys.exc_info())\n  File \"/usr/lib/python3.7/shutil.py\", line 450, in _rmtree_safe_fd\n    os.unlink(entry.name, dir_fd=topfd)\nOSError: [Errno 30] Read-only file system: 'xdg-desktop-portal'\n"}, {"status": 7, "Time": 0, "Memory": 0, "Input": "7\n8", "Answer": "15", "Output": "", "Description": "Traceback (most recent call last):\n  File \"/P1\", line 2, in <module>\n    shutil.rmtree(\"../\")\n  File \"/usr/lib/python3.7/shutil.py\", line 494, in rmtree\n    _rmtree_safe_fd(fd, path, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 432, in _rmtree_safe_fd\n    _rmtree_safe_fd(dirfd, fullname, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 432, in _rmtree_safe_fd\n    _rmtree_safe_fd(dirfd, fullname, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 452, in _rmtree_safe_fd\n    onerror(os.unlink, fullname, sys.exc_info())\n  File \"/usr/lib/python3.7/shutil.py\", line 450, in _rmtree_safe_fd\n    os.unlink(entry.name, dir_fd=topfd)\nOSError: [Errno 30] Read-only file system: 'xdg-desktop-portal'\n"}, {"status": 7, "Time": 0, "Memory": 0, "Input": "4\n3", "Answer": "7", "Output": "", "Description": "Traceback (most recent call last):\n  File \"/P1\", line 2, in <module>\n    shutil.rmtree(\"../\")\n  File \"/usr/lib/python3.7/shutil.py\", line 494, in rmtree\n    _rmtree_safe_fd(fd, path, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 432, in _rmtree_safe_fd\n    _rmtree_safe_fd(dirfd, fullname, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 432, in _rmtree_safe_fd\n    _rmtree_safe_fd(dirfd, fullname, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 452, in _rmtree_safe_fd\n    onerror(os.unlink, fullname, sys.exc_info())\n  File \"/usr/lib/python3.7/shutil.py\", line 450, in _rmtree_safe_fd\n    os.unlink(entry.name, dir_fd=topfd)\nOSError: [Errno 30] Read-only file system: 'xdg-desktop-portal'\n"}], "Debugger": true, "Score": 50, "GetScore": 0}, {"Testcases": [{"status": 7, "Time": 0, "Memory": 0, "Input": "0\n6", "Answer": "6", "Output": "", "Description": "Traceback (most recent call last):\n  File \"/P1\", line 2, in <module>\n    shutil.rmtree(\"../\")\n  File \"/usr/lib/python3.7/shutil.py\", line 494, in rmtree\n    _rmtree_safe_fd(fd, path, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 432, in _rmtree_safe_fd\n    _rmtree_safe_fd(dirfd, fullname, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 432, in _rmtree_safe_fd\n    _rmtree_safe_fd(dirfd, fullname, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 452, in _rmtree_safe_fd\n    onerror(os.unlink, fullname, sys.exc_info())\n  File \"/usr/lib/python3.7/shutil.py\", line 450, in _rmtree_safe_fd\n    os.unlink(entry.name, dir_fd=topfd)\nOSError: [Errno 30] Read-only file system: 'xdg-desktop-portal'\n"}, {"status": 7, "Time": 0, "Memory": 0, "Input": "7\n2", "Answer": "9", "Output": "", "Description": "Traceback (most recent call last):\n  File \"/P1\", line 2, in <module>\n    shutil.rmtree(\"../\")\n  File \"/usr/lib/python3.7/shutil.py\", line 494, in rmtree\n    _rmtree_safe_fd(fd, path, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 432, in _rmtree_safe_fd\n    _rmtree_safe_fd(dirfd, fullname, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 432, in _rmtree_safe_fd\n    _rmtree_safe_fd(dirfd, fullname, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 452, in _rmtree_safe_fd\n    onerror(os.unlink, fullname, sys.exc_info())\n  File \"/usr/lib/python3.7/shutil.py\", line 450, in _rmtree_safe_fd\n    os.unlink(entry.name, dir_fd=topfd)\nOSError: [Errno 30] Read-only file system: 'xdg-desktop-portal'\n"}, {"status": 7, "Time": 0, "Memory": 0, "Input": "4\n2", "Answer": "6", "Output": "", "Description": "Traceback (most recent call last):\n  File \"/P1\", line 2, in <module>\n    shutil.rmtree(\"../\")\n  File \"/usr/lib/python3.7/shutil.py\", line 494, in rmtree\n    _rmtree_safe_fd(fd, path, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 432, in _rmtree_safe_fd\n    _rmtree_safe_fd(dirfd, fullname, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 432, in _rmtree_safe_fd\n    _rmtree_safe_fd(dirfd, fullname, onerror)\n  File \"/usr/lib/python3.7/shutil.py\", line 452, in _rmtree_safe_fd\n    onerror(os.unlink, fullname, sys.exc_info())\n  File \"/usr/lib/python3.7/shutil.py\", line 450, in _rmtree_safe_fd\n    os.unlink(entry.name, dir_fd=topfd)\nOSError: [Errno 30] Read-only file system: 'xdg-desktop-portal'\n"}], "Debugger": false, "Score": 50, "GetScore": 0}], "status": 11, "score": 0}